File: src/main/java/com/bdev/hogwarts_api/dao/EventParticipantDao.java
package com.bdev.hogwarts_api.dao;

import com.bdev.hogwarts_api.data.model.event_participant.EventParticipantModel;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface EventParticipantDao extends JpaRepository<EventParticipantModel, Long> {
    List<EventParticipantModel> getAllByEventId(long eventId);
}

File: src/main/java/com/bdev/hogwarts_api/dao/EventParticipantDao.kt
package com.bdev.hogwarts_api.dao

import com.bdev.hogwarts_api.data.model.event_participant.EventParticipantModel
import org.springframework.data.jpa.repository.JpaRepository

interface EventParticipantDao : JpaRepository<EventParticipantModel, Long> {
    fun getAllByEventId(eventId: Long): List<EventParticipantModel>
}
